test/puppetlabs/puppetdb/query/environments_test.clj

```diff
@@ -18,19 +23,19 @@
     (is (= #{{:name "foo"}
              {:name "bar"}
              {:name "baz"}}
-           (set (:result (query-environments :v4 (query->sql :v4 nil))))))))
+           (set (query-environments :v4))))))
 
 (def jsonify (comp json/parse-strict-string json/generate-string))
```

Comment:
what a strange thing to do. in any case, the name makes it sound like it's just generate-string. json-round-trip or something would be better.